package.json

```diff
@@ -43,62 +43,63 @@
-    "@storybook/react": "7.5.3",
-    "@storybook/react-vite": "7.5.3",
-    "@storybook/testing-library": "0.2.2",
-    "@storybook/theming": "7.5.3",
-    "@swc/core": "1.3.99",
-    "@swc/jest": "0.2.29",
-    "@testing-library/dom": "9.3.3",
+    "@formatjs/cli": "6.2.7",
+    "@storybook/addon-a11y": "8.0.4",
+    "@storybook/addon-essentials": "8.0.4",
+    "@storybook/addon-interactions": "8.0.4",
+    "@storybook/addon-links": "8.0.4",
+    "@storybook/addon-themes": "8.0.4",
+    "@storybook/blocks": "8.0.4",
+    "@storybook/manager-api": "8.0.4",
+    "@storybook/react": "8.0.4",
+    "@storybook/react-vite": "8.0.4",
+    "@storybook/test": "8.0.4",
+    "@storybook/theming": "8.0.4",
+    "@swc/core": "1.4.8",
+    "@swc/jest": "0.2.36",
+    "@testing-library/dom": "9.3.4",
     "@testing-library/jest-dom": "6.1.4",
     "@testing-library/react": "12.1.5",
-    "@testing-library/user-event": "14.5.1",
-    "@tokens-studio/sd-transforms": "0.11.10",
-    "@types/eslint": "8.44.7",
-    "@types/jest": "29.5.10",
-    "@types/jest-axe": "3.5.8",
+    "@testing-library/user-event": "14.5.2",
+    "@tokens-studio/sd-transforms": "0.14.4",
+    "@types/eslint": "8.56.6",
+    "@types/jest": "29.5.12",
+    "@types/jest-axe": "3.5.9",
     "@types/node": "20.11.30",
-    "@types/react": "17.0.70",
-    "@types/react-dom": "17.0.23",
-    "@typescript-eslint/eslint-plugin": "6.12.0",
-    "@typescript-eslint/parser": "6.12.0",
-    "@vitejs/plugin-react": "4.2.0",
+    "@types/react": "17.0.79",
+    "@types/react-dom": "17.0.25",
+    "@typescript-eslint/eslint-plugin": "7.3.1",
+    "@typescript-eslint/parser": "7.3.1",
+    "@vitejs/plugin-react": "4.2.1",
     "classnames": "2.5.1",
-    "eslint": "8.54.0",
+    "eslint": "8.57.0",
     "eslint-config-sonarqube": "2.1.3",
     "eslint-plugin-header": "3.1.1",
-    "eslint-plugin-import": "2.29.0",
-    "eslint-plugin-jest": "27.6.0",
-    "eslint-plugin-jest-dom": "5.1.0",
+    "eslint-plugin-import": "2.29.1",
+    "eslint-plugin-jest": "27.9.0",
+    "eslint-plugin-jest-dom": "5.2.0",
     "eslint-plugin-jsx-a11y": "6.8.0",
     "eslint-plugin-promise": "6.1.1",
-    "eslint-plugin-react": "7.33.2",
+    "eslint-plugin-react": "7.34.1",
     "eslint-plugin-react-hooks": "4.6.0",
     "eslint-plugin-testing-library": "5.11.1",
     "jest": "29.7.0",
     "jest-axe": "8.0.0",
     "jest-environment-jsdom": "29.7.0",
-    "prettier": "3.1.0",
+    "prettier": "3.2.5",
     "react": "17.0.2",
     "react-dom": "17.0.2",
-    "react-intl": "6.5.5",
-    "react-router-dom": "6.11.2",
-    "rollup-plugin-license": "3.2.0",
-    "storybook": "7.5.3",
-    "style-dictionary": "3.9.0",
-    "typescript": "5.2.2",
-    "vite": "5.0.0",
-    "vite-plugin-dts": "3.6.3"
+    "react-intl": "6.6.2",
+    "react-router-dom": "6.22.3",
+    "rollup": "4.13.0",
+    "rollup-plugin-license": "3.3.1",
+    "storybook": "8.0.4",
+    "style-dictionary": "3.9.2",
+    "typescript": "5.3.3",
```

Comment:
The last version is 5.4.3, but it's generating tons of errors on every file like if the whole react typing was missing. It's probably a conflict or incompatibility, I tried to resolve it while keeping react 17 types, but didn't manage so far.

Reply:
I'm not surprised

Reply:
Why? Is there something specific about this version?